Program.cs

```diff
@@ -3,17 +3,60 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to swap the indices of a given list of numbers
+// A random series of numbers is displayed to the user and the user is asked to provide the two indices to be swapped.
+// The number series after swapping the indices is displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap Indices</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method prints a series of numbers based on the count given by the user and displays the updated series after swapping the user asked indices</summary>
+      static void Main () {
+         Console.Write ("Enter the count (2-10) of random numbers to be displayed: ");
+         int count = GetValidNumber ();
+         if (count < 2 || count > 10) {
+            Console.Write ("Count must be between 2 to 10.");
+            return;
+         }
+         int[] numberSeries = new int[count];
+         for (int i = 0; i < count; i++) { //Creates a random number series of given count.
+            Random random = new ();
+            numberSeries[i] = random.Next (1, 10);
+         }
+         Console.Write ("Before Swapping: ");
+         foreach (int number in numberSeries) Console.Write (number + " ");
+
+         Console.Write ("\nEnter the indices to be swapped: \nFirst Index: ");

```

Comment:
When asking the user to enter indices, it should be clear to them that you are asking for a zero based index.